File: ATG/app/src/main/java/com/kuhakupixel/atg/backend/ScanProgressData.java
package com.kuhakupixel.atg.backend;


public class ScanProgressData {
    public Integer current;
    public Integer max;
    public Boolean is_finished;

    public ScanProgressData(Integer current, Integer max, Boolean is_finished) {
        this.current = current;
        this.max = max;
        this.is_finished = is_finished;
    }
}

File: ATG/app/src/main/java/com/kuhakupixel/atg/backend/ScanProgressData.kt
package com.kuhakupixel.atg.backend

class ScanProgressData(current: Int, max: Int, is_finished: Boolean) {
    var current: Int
    var max: Int
    var is_finished: Boolean

    init {
        this.current = current
        this.max = max
        this.is_finished = is_finished
    }
}